frontend/src/components/BCNavbar/BCNavbar.stories.jsx

```diff
@@ -93,7 +93,12 @@ const Template = (args) => <><AppNavbar {...args} />
 // Define the different stories
 export const Default = Template.bind({});
 Default.args = {
-  title: 'Low Carbon Fuel Standard',
-  organizationName: 'BC Government',
-  balance: '50,000',
+  routes: [
+    { icon: 'home', name: 'Dashboard', route: '/' },
+    { icon: 'folder', name: 'document', route: '/document' },
+    { icon: 'account_balance', name: 'Transactions', route: '/transactions' },
```

Comment:
Double check the mockups from Al. I don't believe we have icons in the navbar.

Reply:
Reviewed with Al, and removed icons where ever applicable. But in the generic component have left the option if needed.